hw8/task_6.py

```diff
@@ -0,0 +1,27 @@
+# Напишите функцию, которая преобразует pickle файл
+# хранящий список словарей в табличный csv файл.
+# Для тестированию возьмите pickle версию файла из задачи
+# 4 этого семинара.
+# Функция должна извлекать ключи словаря для заголовков
+# столбца из переданного файла.
+
+import pickle
+import csv
+
+
+__all__ = ['pickle_to_csv']
+
+
+def pickle_to_csv(pickle_file, csv_file):
+    with open(pickle_file, 'rb',) as pickle_f:
+        list_of_dict = [item for item in pickle.load(pickle_f).values()]
+        headers = list(list_of_dict[0].keys())
+    with open(csv_file, 'w', newline='', encoding='utf-8') as csv_f:
+        csv_write = csv.DictWriter(csv_f, fieldnames=headers, dialect='excel')
```

Comment:
Как я понял у вас список список словаре, тогда можно сделать так
```suggestion
        list_of_dict = pickle.load(pickle_f)
    with open(csv_file, 'w', newline='', encoding='utf-8') as csv_f:
        csv_write = csv.DictWriter(csv_f, fieldnames=list_of_dict[0], dialect='excel')
```